server/repository/src/db_diesel/assets/asset_catalogue_item_row.rs

```diff
@@ -101,9 +93,10 @@ impl<'a> AssetCatalogueItemRowRepository<'a> {
     }
 
     pub fn delete(&self, asset_catalogue_item_id: &str) -> Result<(), RepositoryError> {
```

Comment:
Should we call these `soft_delete()`?

Reply:
I used `mark_deleted` in HSH I think.
I do like the idea of having a slightly different name but want to be consistent too (Where possible)

Reply:
`mark_deleted` sounds good to me.

I was working on name merge yesterday and there was also a currency soft delete earlier by @roxy-dao